npm/pkg/dataplane/ipsets/ipsetmanager_linux.go

```diff
@@ -774,6 +774,17 @@ func (iMgr *IPSetManager) deleteMemberForApply(creator *ioutil.FileCreator, set
 			},
 		},
 	}
+
+	splitMember := strings.Split(member, space)
```

Comment:
`[`strings.Fields`](https://pkg.go.dev/strings#Fields) is slightly better for this purpose since it considers all manner of what could constitute a "space". Granted, this is *probably* fine here given that `space` is likely the only one we'll encounter here. Just mentioning it for awareness more than anything.

Reply:
agh good call out, but ya for this use case there can only be a single space, added here:
https://github.com/Azure/azure-container-networking/blob/b29bdac938d49872cf19f383a65f325546612d12/npm/pkg/controlplane/translation/translatePolicy.go#L150